### DISCUSSION TITLE
New to GitHub IDE

### DISCUSSION THREAD

**Discussion by VikramS23** [community/community#145346] [Codespaces] [answered]
### Select Topic Area

General

### Body

Hi, I am new to a GitHub IDE. I need to do Java Development in a GitHub IDE, is GitHub Codespaces what I need to use? Can Maven be installed in the IDE as well?

Thank you.

**Comment by toygma** [comment, marked as answer]
On GitHub Codespaces, you can install Maven directly from the terminal. If you want to add the Maven plugin for VS Code, you can search for and install plugins like "Maven for Java" from the "Extensions" menu.

https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven

**Comment by VikramS23** [comment]
Thank you, this is very helpful.

**Reply by toygma** [reply]
You're welcome
